services/cdnjs/cdnjs.js

```diff
@@ -7,6 +7,11 @@ const { version: versionColor} = require('../../lib/color-formatters');
 
 module.exports = class Cdnjs extends BaseJsonService {
   async handle({library}) {
```

Comment:
Could you move this function declaration down? I'd prefer to declare the helper functions first, then the function that calls them. I've seen lots of work done with both styles, and find that way to be faster to read.